package.json

```diff
@@ -43,7 +43,7 @@
     "@commitlint/config-conventional": "^17.6.6",
```

Comment:
I'm just making this into a draft for a while, because I just checked if lime-webclient gets any build errors with this version, and it _doesn't_, which is probably fine, but I'm just a little worried that we might have problems with stuff not working, so I'm going to test it first 😅

Reply:
Doesn't seem to be any problems 👍